src/main/java/business/controllers/LabRequestController.java

```diff
@@ -242,12 +240,11 @@ public LabRequestRepresentation unapprove(UserAuthenticationToken user,
     }
 
     @PreAuthorize("isAuthenticated() and hasPermission(#id, 'labRequestAssignedToUser')")
-    @RequestMapping(value = "/labrequests/{id}/sending", method = RequestMethod.PUT)
+    @RequestMapping(value = "/labrequests/{id}/sending", method = RequestMethod.PUT, consumes = {"application/json"})
     public LabRequestRepresentation sending(UserAuthenticationToken user,
             @PathVariable Long id,
-            @RequestBody LabRequestRepresentation body) {
+            @RequestBody String body) {
```

Comment:
Isn't it possible to use `Date` here or an object containing a date and either have custom (de)serialisers or use a timestamp? I know that serialising dates can be quite tricky (with time zones, lack of a single format), but passing a string and 'manually' deserialising is so much different from the other endpoints...